From e7b3803b7bd2b78ac392e9df15762e10844f0fa3 Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 24 Apr 2023 21:50:12 +0100
Subject: [PATCH 2/3] gnu: cl-fast-ecs: Update to 0.2.2.

* gnu/packages/lisp-xyz.scm (sbcl-cl-fast-ecs): Update to 0.2.2.
[native-inputs]: Add sbcl-chlorophyll, sbcl-cl-mock.
---
 gnu/packages/lisp-xyz.scm | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/lisp-xyz.scm b/gnu/packages/lisp-xyz.scm
index fcc549d632..dbf4ea87bd 100644
--- a/gnu/packages/lisp-xyz.scm
+++ b/gnu/packages/lisp-xyz.scm
@@ -7049,7 +7049,7 @@ (define-public ecl-cl-fastcgi
 (define-public sbcl-cl-fast-ecs
   (package
     (name "sbcl-cl-fast-ecs")
-    (version "0.1.1")
+    (version "0.2.2")
     (source
      (origin
        (method git-fetch)
@@ -7058,10 +7058,10 @@ (define-public sbcl-cl-fast-ecs
              (commit version)))
        (file-name (git-file-name "cl-fast-ecs" version))
        (sha256
-        (base32 "06cnhm8zpyqyjr17mji5wvj4gh2glpdw8gqy1vwrq3vgphfmg560"))))
+        (base32 "00nw5nwzcz8x1x1lycmjik8pcqzxrl896j0xjjl33rjljsmj45sx"))))
     (build-system asdf-build-system/sbcl)
     (native-inputs
-     (list sbcl-parachute))
+     (list sbcl-chlorophyll sbcl-cl-mock sbcl-parachute))
     (inputs
      (list sbcl-alexandria sbcl-trivial-garbage))
     (home-page "https://lockie.gitlab.io/cl-fast-ecs/")
-- 
2.39.2

